package.json

```diff
@@ -19,7 +21,8 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public"
+    "build-storybook": "build-storybook -s public",
+    "publish" : "aws s3 cp --recursive ./build s3://streetlives-web/ --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers"
```

Comment:
@jbeard4 Just to make sure: Versioning is enabled on this bucket, right?

Reply:
I haven't enabled it. Do you think I should?

Reply:
If this is the bucket we intend to use for actual deployment of the app bundle, I think we should. That way we can easily rollback to a known working version if something fails.

Reply:
OK, I'll add versioning. Thanks.